extensions/amp-story/1.0/live-story-manager.js

```diff
@@ -64,37 +69,28 @@ export class LiveStoryManager {
       'amp-story must contain id to use the live story functionality'
     );
 
+    this.ampStory_.element.addEventListener(EventType.STORY_LOADED, () => {
+      Services.extensionsFor(this.ampdoc_.win).installExtensionForDoc(
+        this.ampdoc_,
+        'amp-live-list'
+      );
+    });
```

Comment:
Can we also append the liveList once the story is loaded? It shouldn't make any difference but might protect us from some timeouts.